ang/crmMailing/BlockMailing.html

```diff
@@ -79,5 +79,19 @@
         placeholder="Subject"
         name="subject" />
     </div>
+    <div ng-if="crmMailingConst.isMultiLingual">
+      <div crm-ui-field="{name: 'subform.language', title: ts('Language')}">
+        <select
+          crm-ui-id="subform.language"
+          crm-ui-select="{dropdownAutoWidth : true, allowClear: false}"
+          name="language"
+          ng-model="mailing.language"
+          required
+          >
+          <option value=""></option>
```

Comment:
I think we need to remove ```required``` property here https://github.com/civicrm/civicrm-core/pull/9152/files#diff-45957c60f2f9f6522bab9138451ac71dR89